frontend/packages/ceph-storage-plugin/src/components/create-storage-system/reducer.ts

```diff
@@ -125,12 +121,80 @@ export type LocalVolumeSet = {
   chartNodes: Set<string>;
 };
 
+const setDeployment = (
+  prevState: WizardState,
```

Comment:
```suggestion
```
I don't see this required at all. Use `Object.assign{}` to override certain properties.

Reply:
Yeah, I just though I wont confuse for readability, but tahnks for second opinion